src/Artsy/index.tsx

```diff
@@ -1,7 +1,2 @@
 export * from "./Analytics"
-export {
-  ContextProvider,
-  ContextConsumer,
-  SystemContext,
-  ContextProps,
-} from "./SystemContext"
+export * from "./SystemContext"
```

Comment:
This _may_ also fix the build issues on Netlify. For some reason we were getting some odd circular-ref like issues when compiling that are fixed when we export named exports in batch. (This will also fix the warning that appears in the console when booting Storybooks.)